packages/devtools-reps/.eslintrc.js

```diff
@@ -0,0 +1,481 @@
+"use strict";
```

Comment:
After landing we should compare this to devtools-core's .eslintrc, fix up the differences, and then (probably) just remove this file.